content-repo/extra-docs/releases/20.11.1.md

```diff
@@ -0,0 +1,1116 @@
+# Cortex XSOAR Content Release Notes for version 20.11.1 (196572)
+##### Published on 22 November 2020
+
+### Breaking Changes
+Several packs include breaking changes.
+
+- [CrowdStrike Malquery Pack v1.0.1](#crowdstrike-malquer-pack-v101)
+- [PAN-OS Pack v1.6.6](#pan-os-pack-v166)
+- [PassiveTotal Pack v2.0.5](#passivetotal-pack-v205-partner-supported)
+
+### New: Asset Pack v1.0.0
+#### Incident Fields
+- **Asset Table**
+A table view of all assets related to the offense.
+- **Description - Asset**
+The description of the asset.
+- **ID - Asset**
+The ID of the asset.
+- **IP Address - Asset**
+The IP address provided for the asset.
+- **Location - Asset**
+The location of the asset.
+- **MAC Address - Asset**
+The MAC address provided for the asset.
+- **Switch ID - Asset**
+The ID of the switch the the asset is connected to.
+- **Switch Port ID - Asset**
+The ID of the switch port for which the asset is connected to.
+
+
+---
+
+### New: Cisco ESA IronPort Email API Pack v1.0.0
```

Comment:
Add community contributed. @yaakovi aren't we adding `(Community Contributed)` automatically?
```suggestion
### New: Cisco ESA IronPort Email API Pack v1.0.0 (Community Contributed)
```